src/components/calcite-popover/calcite-popover.scss

```diff
@@ -63,11 +63,11 @@
 }
 
 .content {
-  @apply flex
-  flex-col
+  @apply flex-col
   flex-no-wrap
   self-center
   h-full;
+  flex: 1 0 auto;
```

Comment:
What do you think about using `w-full` instead?

Reply:
That works as well, is there a benefit to using one or the other?